src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityOperatorBase.cs

```diff
@@ -54,13 +61,14 @@ protected sealed override (TypeWithAnnotations ReturnType, ImmutableArray<Parame
         {
             var compilation = DeclaringCompilation;
             var location = ReturnTypeLocation;
+            var annotation = ContainingType.IsRecordStruct ? NullableAnnotation.Oblivious : NullableAnnotation.Annotated;
```

Comment:
What's the effect of using Oblivious here vs NotAnnotated?

Reply:
There's no effect because `TypeWithAnnotation` already has special case for structs, but it seemed misleading to call the API with NotAnnotated unconditionally.